worlds/factorio/Locations.py

```diff
@@ -1,32 +1,30 @@
 from typing import Dict, List
 
-from .Technologies import factorio_base_id, factorio_id
+from .Technologies import factorio_base_id
 from .Options import MaxSciencePack
 
 boundary: int = 0xff
 total_locations: int = 0xff
 
 assert total_locations <= boundary
-assert factorio_base_id != factorio_id
 
 
 def make_pools() -> Dict[str, List[str]]:
     pools: Dict[str, List[str]] = {}
     for i, pack in enumerate(MaxSciencePack.get_ordered_science_packs(), start=1):
-        max_needed: int = sum(divmod(boundary, i))
-        scale: float = boundary / max_needed
+        max_needed: int = 0xff
         prefix: str = f"AP-{i}-"
-        pools[pack] = [prefix + hex(int(x * scale))[2:].upper().zfill(2) for x in range(1, max_needed + 1)]
+        pools[pack] = [prefix + hex(int(x))[2:].upper().zfill(2) for x in range(1, max_needed + 1)]
```

Comment:
x is already an int.
```suggestion
        pools[pack] = [prefix + hex(x)[2:].upper().zfill(2) for x in range(1, max_needed + 1)]
```